Doc/howto/argparse.rst

```diff
@@ -788,6 +788,59 @@ but not both at the same time:
      -q, --quiet
 
 
+Internationalization support
+============================
+
+The output of the :mod:`argparse` module such as its help text and error
+messages are all made translatable using the :mod:`gettext` module. This
+allows applications to easily localize messages produced by
+:mod:`argparse`. (If you are not familiar with Internationalization and
+Localization, you can check out :ref:`i18n-howto`).
+
+For instance, in this :mod:`argparse` output:
+
+.. code-block:: shell-session
+
+   $ python prog.py --help
+   usage: prog.py [-h] [-v | -q] x y
+
+   calculate X to the power of Y
+
+   positional arguments:
+     x              the base
+     y              the exponent
+
+   options:
+     -h, --help     show this help message and exit
+     -v, --verbose
+     -q, --quiet
+
+The strings ``usage:``, ``positional arguments:``, ``options:`` and
+``show this help message and exit`` are all translatable.
+
+In order to translate these strings, you will probably want to extract them
```

Comment:
I am not keen on "probably". It introduces doubt and puts a burden on the reader. Tell them if they should (or must). You could say "do xxx, unless yyy."